tap-lightcast/meltano.yml

```diff
@@ -0,0 +1,25 @@
+version: 1
+send_anonymous_usage_stats: false
+project_id: 4935d052-aee1-46fa-89f0-a94cdc080064
+plugins:
+  extractors:
+  - name: tap-lightcast
+    namespace: tap-lightcast
+    executable: ./tap-lightcast.sh
+    capabilities:
+    - state
+    - catalog
+    - discover
+    config:
+      client_id: CLIENT-ID
+      client_secret: CLIENT-SECRET
```

Comment:
I did not include limit here and the extractor failed.  Can you either add the limit config as required or update so that it works without including it?